spec/fixtures/http.ts

```diff
@@ -39,15 +39,15 @@ export function mockRCVariableFetch(
   data: any,
   token: string = 'thetoken'
 ): nock.Scope {
-  let mock: nock.Scope = nock('https://runtimeconfig.googleapis.com').get(
+  let interceptor = nock('https://runtimeconfig.googleapis.com').get(
     `/v1beta1/projects/${projectId}/configs/firebase/variables/${varName}`
   );
 
   if (token) {
-    mock = mock.matchHeader('Authorization', `Bearer ${token}`);
+    interceptor = interceptor.matchHeader('Authorization', `Bearer ${token}`);
```

Comment:
well, since you're changing the name here...

`token` has a default value in the function signature. This `if` is unnecessary.

Reply:
Nice catch, done in 76ce309dd2c2dc1b03f3f6bdd0c2dbcc7c5f2640